traffic_portal/app/src/modules/private/cdns/config/ConfigController.js

```diff
@@ -41,153 +37,100 @@ var ConfigController = function(cdn, currentSnapshot, newSnapshot, $scope, $stat
 	var oldEdgeCacheGroups = currentSnapshot.edgeLocations,
 		newEdgeCacheGroups = newSnapshot.edgeLocations;
 
+	var oldTrafficRouterCacheGroups = currentSnapshot.trafficRouterLocations,
```

Comment:
Can you use `let`/`const` for block-scope variables? Globals can use `var` for all I care, though. With a wacky webpack-and-turning-html-into-javascript-strings build system there's no telling what that would do to global-scope variables.

Reply:
makes sense. this could really be done throughout the whole code base. you cant to create an issue for that a we can mark it as tech debt?

Reply:
I think there is an issue. If not, I'll open one.